### ISSUE TITLE
Documentation around project documentation is lacking

### ISSUE THREAD

**Issue by ryanjm** [realm/jazzy#555]
I want to generate documentation for my Objective-C / Swift app. It isn't a framework / module. The documentation isn't very clear on how to properly setup the command to run in this case (or if it is even supported).

The first issue appears to be around `-x`. The `-h` says this:

```
-x arg1,arg2,…argN,              Arguments to forward to xcodebuild
```

I think it would be clearer to say `arg1,arg1Value,arg2,arg2Value` unless I'm miss using it.

I've gotten this far:

```
jazzy -c -o ./docs --min-acl internal -x -workspace,MyProject.xcworkspace,-scheme,Debug
```

And after spitting out a lot of data about the build itself it says:

> ** BUILD SUCCEEDED **
> 
> 
> Please pass in xcodebuild arguments using -x
> If build arguments are correct, please file an issue on https://github.com/realm/jazzy/issues

And no documentation is generated.

When I try to add `--objc` I get:

```
% jazzy --objc -c -o ./docs --min-acl internal -x -workspace,MyProject.xcworkspace,-scheme,Debug
/Users/ryanjm/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/jazzy-0.0.20/lib/jazzy/config.rb:142:in `parse!': invalid option: --objc (OptionParser::InvalidOption)
	from /Users/ryanjm/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/jazzy-0.0.20/bin/jazzy:15:in `<top (required)>'
	from /Users/ryanjm/.rbenv/versions/2.1.2/bin/jazzy:23:in `load'
	from /Users/ryanjm/.rbenv/versions/2.1.2/bin/jazzy:23:in `<main>'
```

What am I missing?

**Comment by jpsim** [comment]
you have a very old version of jazzy installed (0.0.20). Please install the latest version (0.6.0). And I agree, we certainly need to improve jazzy's own usage documentation, and are tracking that as one of the oldest open tickets 😱  (#77)

**Comment by ryanjm** [comment]
It looks like 0.6.0 isn't compatible with `cocoapods` `1.0.0`.

```
jazzy (= 0.6) was resolved to 0.6.0, which depends on
      cocoapods (~> 0.39)
```

Looks like that is being handled with #568.
